guides/use-microprofile-lra.adoc

```diff
@@ -0,0 +1,165 @@
+    LOG.info("Complete ID = " + lraId);
+    ...
+----
+
+Or, in the case of LRA cancel, the compensation callback will be invoked instead:
+
+[source,java]
+----
+@Compensate
+@PUT
+@Path("/compensate")
+public Response compensate(@HeaderParam(LRA.LRA_HTTP_CONTEXT_HEADER) String lraId) {
+    LOG.info("Compensate ID = " + lraId);
+    ...
+----
+
+The full example is available at https://github.com/xstefank/lra-wildfly-example.
+
+If you deploy this application to WildFly (28.0.0+) with both microprofile-lra-coordinator and
+microprofile-lra-participant subsystems enabled, you can make the following HTTP invocation to see how the coordinator
+invokes the complete callbacks or the compensation callbacks of the two defined participants:
+
+[source,bash]
+----
+$ curl localhost:8080/lra-participant/lra-participant-1/doInLRA
+
+# in WFLY console log
+15:14:50,128 INFO  [io.xstefank.LRAParticipant1] (default task-1) Work LRA ID = http://localhost:8080/lra-coordinator/lra-coordinator/0_ffff0aca8851_-3330598e_646cbc18_14
+15:14:50,158 INFO  [io.xstefank.LRAParticipant2] (default task-2) Work LRA ID = http://localhost:8080/lra-coordinator/lra-coordinator/0_ffff0aca8851_-3330598e_646cbc18_14
+15:14:50,183 INFO  [io.xstefank.LRAParticipant1] (default task-3) Complete ID = http://localhost:8080/lra-coordinator/lra-coordinator/0_ffff0aca8851_-3330598e_646cbc18_14
+15:14:50,191 INFO  [io.xstefank.LRAParticipant2] (default task-3) Complete ID = http://localhost:8080/lra-coordinator/lra-coordinator/0_ffff0aca8851_-3330598e_646cbc18_14
+
+
+$ curl "localhost:8080/lra-participant/lra-participant-1/doInLRA?fail=true"
+
+# in WFLY console log
+15:15:33,516 INFO  [io.xstefank.LRAParticipant1] (default task-1) Work LRA ID = http://localhost:8080/lra-coordinator/lra-coordinator/0_ffff0aca8851_-3330598e_646cbc18_1c
+15:15:33,531 INFO  [io.xstefank.LRAParticipant2] (default task-2) Work LRA ID = http://localhost:8080/lra-coordinator/lra-coordinator/0_ffff0aca8851_-3330598e_646cbc18_1c
+15:15:33,543 INFO  [io.xstefank.LRAParticipant1] (default task-3) Compensate ID = http://localhost:8080/lra-coordinator/lra-coordinator/0_ffff0aca8851_-3330598e_646cbc18_1c
+15:15:33,550 INFO  [io.xstefank.LRAParticipant2] (default task-3) Compensate ID = http://localhost:8080/lra-coordinator/lra-coordinator/0_ffff0aca8851_-3330598e_646cbc18_1c
+----
+
+You can also always check the currently active LRAs with a direct call to the coordinator API:
+
+[source,bash]
+----
+$ curl localhost:8080/lra-coordinator/lra-coordinator
+[]%
+----
+
+== Conclusion
+
+In this post, we showed you how to configure and use the MicroProfile LRA specification in your WildFly applications.
+LRA provides a very broad feature set which we can't cover here. If you are interested in learning more, you can find
+the full specification at https://download.eclipse.org/microprofile/microprofile-lra-2.0/microprofile-lra-spec-2.0.html.
```

Comment:
The References section is missing here.

Reply:
MicroProfile LRA is excluded at the moment. So this is still something TBD.